source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRColumnRenderer.mm

```diff
@@ -0,0 +1,57 @@
+//
+//  ACRColumnRenderer
+//  ACRColumnRenderer.mm
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import "ACRColumnRenderer.h"
+#import "ACRRegistration.h"
+#import "Column.h"
+#import "AdaptiveCard.h"
+
+@implementation ACRColumnRenderer
+
++ (ACRColumnRenderer* ) getInstance
+{
+    static ACRColumnRenderer *singletonInstance = nil;
+    static dispatch_once_t predicate;
+    dispatch_once(&predicate, ^{singletonInstance = [[self alloc] init];});
```

Comment:
There's a shorter way to say this:
`static ARColumnRenderer *singletonInstance = [[self alloc] init];`